hooks.md

```diff
@@ -10,5 +10,5 @@ http://wizard.amigos.institute/ *but should be tracked through bit.ly*
 Hi,
```

Comment:
The `hook.md` file contains the line i copy & paste when i post information about us somewhere on the internet to make them click the link and come to our page. It wont be content on our page